test/Variables/ListVariable.py

```diff
@@ -70,7 +77,7 @@ def check(expected):
 else:
     print('0')
 
-print(" ".join(env['shared']))
+print(",".join(env['shared']))
```

Comment:
Here we print things so that the test "expect" can look for matches - if a space separator is used, we can't distinguish a space-containing value, so had to rework to use a comma.